build.cmd

```diff
@@ -24,9 +24,11 @@ EXIT /b %ERRORLEVEL%
     SET DOTNET_ROOT=%~dp0.dotnet
     :: This tells .NET Core not to go looking for .NET Core in other places
     SET DOTNET_MULTILEVEL_LOOKUP=0
+    SET PATH=%DOTNET_ROOT%;%PATH%
 
-    dotnet dotnet-coverage collect --settings ./eng/CodeCoverage.config --output ./artifacts/TestResults/ "build.cmd -test -bl"
-    dotnet reportgenerator -reports:./artifacts/TestResults/*.cobertura.xml -targetdir:./artifacts/TestResults/CoverageResultsHtml -reporttypes:HtmlInline_AzurePipelines
-    start ./artifacts/TestResults/CoverageResultsHtml/index.html
-    powershell -ExecutionPolicy ByPass -NoProfile -command "./scripts/ValidateProjectCoverage.ps1 -CoberturaReportXml ./artifacts/TestResults/.cobertura.xml"
+    SET TestResultPath=./artifacts/TestResults/Debug/
+    dotnet dotnet-coverage collect --settings ./eng/CodeCoverage.config --output %TestResultPath%/local.cobertura.xml "build.cmd -test -bl"
+    dotnet reportgenerator -reports:%TestResultPath%*.cobertura.xml -targetdir:%TestResultPath%CoverageResultsHtml -reporttypes:HtmlInline_AzurePipelines
+    :: start %TestResultPath%CoverageResultsHtml/index.html
```

Comment:
NIT: Remove commented out code. Also, do we need similar changes on `./build.sh`?

Reply:
This is still WIP.

This functionality needs to be moved out to eng/build.ps1.